port/zos390/omrgetjobname.c

```diff
@@ -52,13 +52,19 @@ omrget_jobname(struct OMRPortLibrary *portLibrary, char *jobname, uintptr_t leng
 	if (tmp_jobname) {
 		memset(tmp_jobname, '\0', J9_MAX_JOBNAME);
 		_JOBNAME(tmp_jobname);  /* requires <31bit address */
+#if !defined(OMR_EBCDIC)
```

Comment:
Why just `!defined(OMR_EBCDIC)` here, but `defined(J9ZOS390) && !defined(OMR_EBCDIC)` elsewhere?

Reply:
given that it is a zos specific file, its safe to assume that `J9ZOS390` is defined.